tests/test_storage.py

```
@@ -0,0 +1,126 @@
+        }
+        tool_fn = _make_tool()
+        result = await tool_fn(action="shares")
+        assert len(result["shares"]) == 2
+
+    async def test_disks(self, _mock_graphql: AsyncMock) -> None:
+        _mock_graphql.return_value = {"disks": [{"id": "d:1", "device": "sda"}]}
+        tool_fn = _make_tool()
+        result = await tool_fn(action="disks")
+        assert len(result["disks"]) == 1
+
+    async def test_disk_details(self, _mock_graphql: AsyncMock) -> None:
```

Ruff (0.15.0)

[warning] Fixture `_mock_graphql` without value is injected as parameter, use `@pytest.mark.usefixtures` instead

(PT019)